testing/testcafe/tests/common/draggable.ts

```diff
@@ -0,0 +1,71 @@
+import { ClientFunction, Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+
+fixture.disablePageReloads`Draggable`
+  .page(url(__dirname, '../container.html'));
+
+test('dxDraggable element should not loose its position on dragging with auto-scroll inside ScrollView (T1169590)', async (t) => {
+  const draggable = Selector('#drag-me');
+
+  await t
+    .drag(draggable, 0, 400, { speed: 0.1 })
+
+    .expect(Selector('.dx-scrollable-container')().scrollTop)
```

Comment:
Let's use instead css class the Scrollable page object model here.

Scrollable page object model here:
testing/testcafe/model/scrollView/scrollable.ts